cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,28 @@
+import { generateUser } from '../support/generate';
+
+Cypress.on('uncaught:exception', () => {
+  return false;
+});
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const { email, username, password } = generateUser();
+
+    cy.visit('/#/register?_k=spysa9');
```

Comment:
It's better to move the base URL ('/#/register?_k=spysa9') to a global configuration file or environment variable. This will make it easier to change the base URL when the test environment changes.